assets/js/src/core/components/grid/grid.tsx

```diff
@@ -197,6 +218,16 @@ export const Grid = ({ enableMultipleRowSelection = false, modifiedCells = [], s
     </div>
   )
 
+  const getContextMenuItems = (row: any): DropdownMenuProps['items'] => {
```

Comment:
This part seems to be not generic enough. Our generic grid component does not know anything about the data, so we must not add logic which is dependent on the data structure here.

Reply:
changed logic with https://github.com/pimcore/studio-ui-bundle/pull/847/commits/d0268e8892180340e516aeafeac15b6aee7f738e